client/js/components/week/week-board.js

```diff
@@ -0,0 +1,21 @@
+/** @jsx React.DOM */
+var React = require('react');
+var WeekDayCard = require('./week-day-card');
+var _ = require('../../utils/utils');
+
+var WeekBoard = React.createClass({
+  render: function(){
+    var weekDayCards = _.map(this.props.data, function(item, key){
```

Comment:
add a more descriptive variable name instead of 'item'. What data is being mapping over?